scripts/00918-AISOSA.php

```diff
@@ -6,7 +6,11 @@
 </head>
 <body>
 <?php
-echo "Hello World!, this is Aisosa Akpevweoghene Ogheneowho Ugono with HNGi7 ID HNG7-00918 using both HTML and PHP for stage 2 task";
+$full_name="Aisosa Akpevweoghene Ogheneowho Ugono";
+$id="HNG-00918";
+$language="PHP";
+$email="akpeugono@gmail.com";
+echo "Hello World!, this is $full_name with HNGi7 ID $id using $language for stage 2 task $email .";
```

Comment:
Please remove the space and fullstop after the email.

Reply:
okay